roles/metallb/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Timeout to wait for MetalLB services to come up
+metal_lb_available_timeout: 120s
+metal_lb_ip_range: "192.168.201.240-192.168.201.250"
```

Comment:
Where did this IP range come from? I can't find a CIDR notation for this range.

Reply:
The IP range is arbitrary, but MetalLB needs a dedicated set of IPs that it can use freely without the help of anything external. These are the IPs assigned to services.

Reply:
> Also you can use something like this
> 
> ```
> # Get the next five addresses using CIDR notation
> # {{ '192.168.1.1/24' | ansible.utils.ipmath(5) }}
> 192.168.1.6
> ```
> 
> to obtain the 240th and 255th IP from a prefix.
> 
> The IP range is set by terraform (for dev) but for prod it should come from somwhere else (terraform).

Feel free to make a code suggestion here.

Reply:
I would appreciate if anyone else wants to take on the metallb stuff, or co-write it with me. I'm just not sure what to use or do here.

Reply:
For now, if you switch to the following or something similar then I think this is kosher enough to merge.

1. Change `metal_lb_ip_range` to a CIDR
2. Make a note in your directions that this'll default to the given CIDR unless overridden
3. Use @muresan 's suggestions for fetching the IPs you need from the CIDR

Reply:
Looking at this, we can use a CIDR for metallb's spec anyway: https://metallb.universe.tf/configuration/_advanced_ipaddresspool_configuration/